product_docs/docs/efm/4/efm_rel_notes/01_efm_410_rel_notes.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Version 4.10"
+---
+
+Enhancements, bug fixes, and other changes in EFM 4.10 include:
+
+| Type | Description |
+| ---- |------------ |
+| Enhancement | Failover Manager was upgraded to use the Bouncy Castle cryptographic library version 1.0.2.5. |
+| Bug Fix | Improve handling of rare case where the standby to promote becomes unavailable during a switchover. [Support ticket: #37266] |
+| Bug Fix | The `efm upgrade-conf` command will not lose .nodes file information when the `-source` and destination directories are the same. [Support ticket: #37479] |
+| Bug Fix | Fixed an issue where the `efm cluster-status` command hid connection errors if every database connection failed. [Support ticket: TBD] |
```

Comment:
@EFM-Bobby, do we have the Support ticket #

Reply:
It's there now (for anyone reading this later)